src/components/global/header/header.tsx

```diff
@@ -57,71 +73,63 @@ export function Header({ showMarquee = true }: { showMarquee?: boolean }) {
                                 >
                                     <img src={peanut_logo.src} alt="logo" className="ml-2 h-6 sm:h-10" />
-                                    <span className="inline lg:px-6">peanut protocol</span>
+                                    <span className="inline sm:px-6">peanut protocol</span>
                                 </div>
                                 <div className="hidden h-full items-center justify-center  sm:flex ">
                                     <div className="flex h-full items-center">
-                                        <Menu as="div" className="relative h-full items-center justify-center ">
-                                            <Menu.Button
-                                                as="a"
-                                                className="flex h-full cursor-pointer items-center px-1 text-base font-bold uppercase text-white no-underline hover:bg-white hover:text-black lg:px-8"
-                                            >
-                                                APP
-                                            </Menu.Button>
-                                            <Transition
-                                                as={Fragment}
-                                                enter="transition ease-out duration-0"
-                                                enterFrom="transform opacity-0 scale-95"
-                                                enterTo="transform opacity-100 scale-100"
-                                                leave="transition ease-in duration-0"
-                                                leaveFrom="transform opacity-100 scale-100"
-                                                leaveTo="transform opacity-0 scale-95"
-                                            >
-                                                <Menu.Items className="absolute left-0 z-10 w-48 origin-top-right bg-black p-0 text-white shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none">
-                                                    {/* <img
-                                                        src={triangle_svg.src}
-                                                        className="absolute -top-4 left-8 h-4 w-4"
-                                                    /> */}
+                                        <Popover className={'h-full'}>
+                                            {({ open }) => (
+                                                <div
+                                                    onMouseEnter={() => onHover(open, 'onMouseEnter')}
+                                                    onMouseLeave={() => onHover(open, 'onMouseLeave')}
+                                                    className="relative h-full items-center justify-center "
+                                                >
+                                                    <Popover.Button
```

Comment:
no idea here, but should abutton be an anchor tag? or btn tag? does it make a difference?

Reply:
yes, fair point. The `as ='a'` is to register that button as an <a> tag. Doesnt need it in this button, since it doesnt have a href value. For buttons that need a href, and need to be treated as a 'a' tag, it does need it 

removed where not needed